### PR TITLE
Refactor interest rate models to share code

### PR DESCRIPTION
**Pull request by jflatow**
Rebased #61 for merging

### CODE REVIEW

File: networks/mainnet.json [hunk 1]
```diff
@@ -63,6 +63,8 @@
         "Pair_ETH_USDC": "0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc",
         "PriceFeedPoster": "0x83563ba7f1b093aae57fe876f8d870f8a1508886",
         "StdComptrollerG5": "0x7b5e3521a049C8fF88e6349f33044c6Cc33c113c"
+        "IRM_USDT_Updateable": "0xFB564da37B41b2F6B6EDcc3e56FbF523bD9F2012",
```
Comment (mykelp) [inline networks/mainnet.json:66]: Looks like the line above is missing a comma

File: networks/mainnet.json [hunk 2]
```diff
@@ -105,6 +107,8 @@
         "PriceData": 10551018,
         "PriceFeed": 10551058,
         "StdComptrollerG5": 10809458
+        "IRM_USDT_Updateable": 10609555,
```
Comment (mykelp) [inline networks/mainnet.json:110]: Line above missing a comma

File: networks/mainnet.json [hunk 3]
```diff
@@ -240,6 +244,8 @@
...
         "StdComptrollerG5": "0x"
+        "IRM_USDT_Updateable":"0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000008e1bc9bf0400000000000000000000000000000000000000000000000000000f207539952d00000000000000000000000000000000000000000000000000000b1a2bc2ec5000000000000000000000000000006d903f6003cca6255d85cca4d3b5e5146dc33925",
```
Comment (mykelp) [inline networks/mainnet.json:247]: Same ^^

Comment (jflatow) [reply networks/mainnet.json:247]: Ah good catch on the manual merge

**Review by mykelp — APPROVED** [review]
👍

**Pull request merged**